yml/hsc.yaml

```diff
@@ -6250,3 +6250,13 @@ tables:
     datatype: double
     description: Scatter (via MAD) of size residual scaled by median size squared
     fits:tunit:
+  - name: psfTraceDelta
+    '@id': '#CcdVisit.psfTraceDelta'
+    datatype: double
+    description: Delta (max - min) of model psf trace values evaluated on a grid of unmasked pixels (pixel)
```

Comment:
Should say "trace radius" here and in `imsim.yaml`, too (see comment in afw PR).